js/test/phone/testnormalize.js

```diff
@@ -961,7 +961,7 @@ module.exports.normalize = {
         test.done();
     },
     
-    testAssistedDialNormalizeBogusNumberForVerizonInvalidCountryCode: function(test) {
+    testAssistedDialNormalizeBogusNumberForVerizonvalidCountryCode: function(test) {
```

Comment:
This test is supposed to test against an invalid country code. Did you meant to change the name of the function?

Reply:
Yes, you are right. it shouldn't be removed for invalid country code testing. I reverted it